org.gnome.meld.json

```diff
@@ -0,0 +1,93 @@
+{
+    "app-id": "org.gnome.meld",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "meld",
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "/man",
+        "/share/man",
+        "/share/gtk-doc",
+        "/share/vala",
+        "*.la",
+        "*.a",
+        "*.pyc",
+        "*.pyo"
+    ],
+    "build-options": {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "V": "1"
+        }
+    },
+    "rename-appdata-file": "meld.appdata.xml",
+    "rename-desktop-file": "meld.desktop",
+    "rename-icon": "meld",
+    "finish-args": [
+        /* X11 + XShm */
+        "--share=ipc", "--socket=x11",
+        /* Wayland */
+        "--socket=wayland",
+        /* Filesystem */
+        "--filesystem=host",
+        /* dconf */
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
```

Comment:
I think you need a little bit more for dconf, see here: https://github.com/flathub/org.gnome.iagno/blob/3c77a388f309ce0df25f12d23463a95edd3279c5/org.gnome.iagno.json#L18

Reply:
Fixed that

Reply:
It had `host` filesystem access so this wasn't need.